src/components/book.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] `required`
- [ ] please enable the linters because it defeats the purpose of installing linters